presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -53,6 +53,17 @@ output data set is not skewed in order to avoid the overhead of hashing and
 redistributing all the data across the network. This can also be specified
 on a per-query basis using the ``redistribute_writes`` session property.
 
+``eager-plan-validation-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Formatting line should be the same number of characters as the line above that it is formatting. 

Local doc build shows the following new warnings (one misformatted line produces two warnings) as follows: 

```
/Users/steveburnett/Documents/GitHub/presto/presto-docs/src/main/sphinx/admin/properties.rst:57: WARNING: Title underline too short.

``eager-plan-validation-enabled``
^^^^^^^^^^^^^^^^^^^^^^^
/Users/steveburnett/Documents/GitHub/presto/presto-docs/src/main/sphinx/admin/properties.rst:57: WARNING: Title underline too short.

``eager-plan-validation-enabled``
^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Thanks @steveburnett for catching that